module.nix

```diff
@@ -121,6 +121,15 @@ in
         description = "QEMU image format to use for the disk images";
         default = "raw";
       };
+
+      useVirtualDevices = lib.mkOption {
```

Comment:
I still quite don't understand why this is needed after partitioning. Are we not using partition labels in any case?

Reply:
There's also `boot.loader.grub.devices`.

Reply:
@Mic92 Thoughts?